src/app/modules/submission/multirespondent-submission/__tests__/multirespondent-submission.service.spec.ts

```diff
@@ -1601,6 +1644,110 @@ describe('multirespondent-submission.service', () => {
         2,
       )
     })
+
+    it('should fire webhook with the correct arguments for first submission', async () => {
+      const encryptedWebhookContent = 'mockEncryptedWebhookContent'
+      // Act
+      await performMultiRespondentPostSubmissionCreateActions({
+        submission: {
+          _id: mockSubmissionId,
+        } as unknown as IMultirespondentSubmissionSchema,
+        submissionId: mockSubmissionId,
+        form: {
+          _id: mockFormId,
+          workflow: singleStepWorkflow,
+          emails: ['email1@example.com'],
+          webhook: { url: MOCK_WEBHOOK_URL, isRetryEnabled: true },
+        } as IPopulatedMultirespondentForm,
+        encryptedPayload: {
+          encryptedContent: 'encryptedContent',
+          version: 1,
+          submissionPublicKey: 'submissionPublicKey',
+          encryptedSubmissionSecretKey: 'encryptedSubmissionSecretKey',
+        } as MultirespondentSubmissionDto,
+        logMeta: {} as any,
+        encryptedWebhookContent,
+      })
+
+      expect(webhookSpy).toHaveBeenCalledWith(
+        expect.objectContaining({
+          _id: mockSubmissionId,
+          encryptedContent: encryptedWebhookContent,
```

Comment:
suggestion: should we also include `encryptedSubmissionSecretKey` in the webhook call assertion as well since it is newly added in this PR?